grakn-graql/src/main/java/ai/grakn/graql/Reasoner.java

```diff
@@ -50,6 +50,14 @@ public Reasoner(GraknGraph graph) {
         linkConceptTypes();
     }
 
+    private void commitGraph() {
+        try {
+            graph.commit();
+        } catch (GraknValidationException e) {
+            LOG.debug(e.getMessage());
```

Comment:
Shouldn't this be `LOG.error` ? I don't expect reasoner to be creating invalid graph mutations so it should definitely be an error. Might even throw the exception as a Runtime one if we are 100% sure reasoner should not be creating invalid commits.